tests/pytests/test_multibyte_char_terms.py

```diff
@@ -0,0 +1,227 @@
+# -*- coding: utf-8 -*-
+
+from common import *
+from RLTest import Env
+import locale
+
+
+def is_locale_available(locale_name):
+    try:
+        locale.setlocale(locale.LC_ALL, locale_name)
+        return True
+    except locale.Error:
+        return False
+
+def testMultibyteChars(env):
+    if not is_locale_available('en_US.UTF-8'):
```

Comment:
I see you add a lot of conditions for the availability of `locale` - why should it not be available?

Reply:
Some docker images don't include it. i.e.: In the CI, the `ubuntu:jammy` image, used for the sanitizer, don't include it.

Reply:
That's a problem. We have to be consistent in our responses, OS-independent.
Can we add it in our installation flow?